lib/private/App/AppManager.php

```diff
@@ -82,6 +83,14 @@ public function __construct(
 	) {
 	}
 
+	private function getNavigationManager(): INavigationManager {
+		if ($this->navigationManager !== null) {
+			return $this->navigationManager;
+		}
+		$this->navigationManager = \OCP\Server::get(INavigationManager::class);
```

Comment:
Could save one line ;)

```suggestion
		if ($this->navigationManager === null) {
			$this->navigationManager = \OCP\Server::get(INavigationManager::class);
		}
```

Reply:
Could save all of them:
```php
return $this->navigationManager ??= \OCP\Server::get(INavigationManager::class);
```

But who’s counting?